packages/tldraw/src/Tldraw.tsx

```diff
@@ -415,7 +435,50 @@ const InnerTldraw = React.memo(function InnerTldraw({
   }, [])
 
   return (
-    <StyledLayout ref={rWrapper} tabIndex={-0} className={settings.isDarkMode ? dark : ''}>
+    <StyledLayout
+      ref={rWrapper}
+      tabIndex={-0}
+      className={settings.isDarkMode ? dark : ''}
+      onDragOver={(e) => e.preventDefault()}
+      onDrop={async (e) => {
```

Comment:
Not sure writing all drag & drop functionality here.

Reply:
Where do you suggest moving it? `<Renderer />` perhaps?

Reply:
I will move it to `TldrawApp` class like all other event handlers on Renderer